cmd/go-ycsb/main.go

```diff
@@ -85,9 +85,18 @@ func initialGlobal(dbName string, onProperties func()) {
 		globalProps = properties.MustLoadFiles(propertyFiles, properties.UTF8, false)
 	}
 
+	propsMap:=map[string][]string{}
 	for _, prop := range propertyValues {
 		seps := strings.SplitN(prop, "=", 2)
-		globalProps.Set(seps[0], seps[1])
+		if propVals,exist:=propsMap[seps[0]];exist{
+			propsMap[seps[0]]=append(propVals,seps[1])
+		}else{
+			propsMap[seps[0]]=[]string{seps[1]}
+		}
+	}
+
+	for key,val:=range propsMap{
+		globalProps.Set(key, strings.Join(val,","))
```

Comment:
seem there is a problem if I pass `-p operationcount=100 -p operationcount=1000.

Reply:
It's really a problem. just I can't find a good solution.
And I don't know why to use "-p", if not, can we avoid these problems?